squash/docs/overview/_index.md

```diff
@@ -212,11 +212,19 @@ You should see a screen similar to the one below.
 
 The Next step is to tell the Squash extension the location of the `squashctl` CLI.
 
-Open **Settings** (File > Preferences > Settings) on Visual Studio Code, and then search for Squash. Once you find Squash, change the value of Path to point to the location of the CLI.
+Open **Settings** (Code > Preferences > Settings) on Visual Studio Code, and then search for Squash. Once you find Squash, change the value of Path to point to the location of the CLI.
 
 ![config](../images/vs-code-config-squash.png)
 
 If the target process was compiled from a different sourcepath, you should also [specify the remote path](/configuration/#source-code-mapping).
+In this quick start, if you choose go example2 app, you have to set “Squash: Remote Path” in VSCode User Settings.
+Please set "/home/yuval/go/src/github.com/solo-io/squash/contrib/example/" to “Squash: Remote Path”.
+
+![vscode-remotepath](../images/vs-code-config-remote-path.png)
+
+If there is an error such as "Error on CreateBreakpoint: could not find /Users/<YourName>/squash/contrib/example/service1/main.go", 
+please correct its remote path in order to match "/home/yuval/go/src/github.com/solo-io/squash/contrib/example/service1/main.go".
```

Comment:
Minor nit. We tend to use markdown backtick \` for most literals in the documentation, right or wrong, so for consistency helpful if you'd follow the same convention.

Reply:
I understood too. 
I will use it from now on.